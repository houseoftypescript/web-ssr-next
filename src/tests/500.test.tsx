import InternalServerErrorPage from '@/pages/500';
import { render } from '@testing-library/react';

describe('InternalServerErrorPage', () => {
  it('to match snapshot', () => {
    const { container } = render(<InternalServerErrorPage />);
    expect(container).toMatchSnapshot();
  });
});
