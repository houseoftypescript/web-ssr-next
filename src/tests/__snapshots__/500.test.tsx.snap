// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InternalServerErrorPage to match snapshot 1`] = `
<div>
  <div
    class="flex flex-col gap-4 items-center justify-center h-screen"
  >
    <h1
      class="text-9xl font-bold"
    >
      500
    </h1>
    <p
      class="text-4xl"
    >
      Internal Server Error
    </p>
  </div>
</div>
`;
